### ISSUE TITLE
Another exception was thrown: Exception: Attempted to reuse a ParagraphRuler but it is currently measuring another paragraph

### ISSUE THREAD

**Issue by AminMemariani** [simpleclub/CaTeX#61]
I have a local JSON file and when I want to show it's content using Catex, it gives me this error. I already posted the complete issue on Stackoverflow here:
[](https://stackoverflow.com/questions/63063855/how-to-fix-another-exception-was-thrown-exception-attempted-to-reuse-a-paragr)

**Comment by creativecreatorormaybenot** [comment]
Thanks for reporting this issue 👍🏽 It seems like a bit of a weird one because CaTeX does not use `Paragraph`s (I think).

Will need to look into this. Can you confirm that this bug must be related to CaTeX and there is no other widget causing your issue?

**Comment by AminMemariani** [comment]
Yes, I replaced the Catex widget with a simple Text and the error is gone.

**Comment by AminMemariani** [comment]
btw, There was an earlier exception in my log which I didn't notice. I updated my post.

**Comment by creativecreatorormaybenot** [comment]
Not sure 😶
